v3-sdk/quoting/src/libs/constants.ts

```diff
@@ -1,26 +1,24 @@
 // This file stores web3 related constants such as addresses, token definitions, ETH currency references and ABI's
 
-import { SupportedChainId, Token } from '@uniswap/sdk-core'
+import { SUPPORTED_CHAINS, Token } from '@uniswap/sdk-core'
 
 // Addresses
 
 export const POOL_FACTORY_CONTRACT_ADDRESS =
   '0x1F98431c8aD98523631AE4a59f267346ea31F984'
-export const QUOTER_CONTRACT_ADDRESS =
-  '0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6'
 
 // Currencies and Tokens
 
 export const WETH_TOKEN = new Token(
-  SupportedChainId.MAINNET,
+  SUPPORTED_CHAINS[0],
```

Comment:
Did you consider importing `ChainId` instead?  Then you can do `ChainId.MAINNET` on line 13 and 21

Reply:
Yeah that would make more sense here.